File: tests/test_src/aplusb.fs
// The entire code is given, you can just review and submit!
open System

[<EntryPoint>]
let main argv =
    let a = Console.ReadLine() |> int
    let b = Console.ReadLine() |> int
    printfn "%d" (a+b)
    0 // return an integer exit code

File: tests/test_src/aplusb.R
# The complete code is given. You can just review and submit!
nums <- read.table("/dev/stdin", sep=" ");
nums <- as.list(as.data.frame(t(nums)))
write.table(as.numeric(nums[1])+as.numeric(nums[2]), sep = "", append=T, row.names = F, col.names = F)
